helm/kiam/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kiam
-version: 6.0.0
+version: 6.0.1
```

Comment:
can you bump to 6.1 as this is not a bug fix

Reply:
Yeah sure. Good point. This should be fixed now.